packages/trigger/schedule-tasks.ts

```diff
@@ -0,0 +1,39 @@
+import { getWorkspacesForSync } from './google-calendar-sync';
+import { schedulableTasksHelper } from './schedule-tasks-helper';
+import { schedules, task } from '@trigger.dev/sdk/v3';
+
+export const scheduleTasksTrigger = schedules.task({
+  id: 'schedule-tasks',
+  cron: {
+    pattern: '15 * * * *',
+  },
+  run: async () => {
+    console.log('=== Starting schedule tasks trigger ===');
+
+    try {
+      // You'll need to specify the workspace ID here
+      // This could come from environment variables or configuration
+      const workspaces = await getWorkspacesForSync();
+
+      console.log(
+        `Found ${workspaces.length} workspaces to sync incrementally`
+      );
+
+      for (const workspace of workspaces) {
+        const ws_id = workspace.ws_id;
+        console.log(`Processing workspace: ${ws_id}`);
+        const result = await schedulableTasksHelper(ws_id);
```

Comment:
please convert this into a trigger task so it can be ran in parallel @phatgg221 @DennieDan